shub_image/utils.py

```diff
@@ -11,6 +13,13 @@
 
 DEFAULT_DOCKER_VERSION = '1.17'
 STATUS_FILE_LOCATION = '.releases'
+_VALIDSPIDERNAME = re.compile('^[a-z0-9][-._a-z0-9]+$', re.I)
+
+
+def debug_log(msg):
+    ctx = click.get_current_context(True)
+    if hasattr(ctx, 'obj') and ctx.obj.get('debug'):
```

Comment:
Isn't `debug` available in [params](http://click.pocoo.org/5/api/#click.Context.params)?